#!/usr/bin/env python
from dynamic_reconfigure.parameter_generator_catkin import *
PACKAGE = "xbox"


gen = ParameterGenerator()

gen.add("V_max", int_t, 0, "Set V_max", 20, 0, 120)
gen.add("W_max", double_t, 1, "Set W_max", 1.57, 0, 6.25)
exit(gen.generate(PACKAGE, "xbox_dynamic_cfg", "param"))
